api/manifest.proto

```diff
@@ -46,34 +46,55 @@ message Flavor {
    map <string, Version> versions = 5;
 }
 
+
+enum Compliance {
+   FIPS_1402 = 0;
+}
+
 message Version {
    // This duplicated in order to make version easier to sort
    string name = 1;
-   // Key is operating system
-   // Note: Keys in proto maps cannot be enums, so this has to be a string.
-   map <string, OperatingSystem> operating_systems = 2; 
-}
-
-message OperatingSystem {
-   // Matches the key, duplicated here to be explicit.
-   OperatingSystemName name = 1;
-   
-   repeated Build builds = 2;
+   repeated Build builds = 2; 
 }
 
 message Build {
-   repeated string operating_system_versions = 1;
+   OperatingSystemFamily operating_system_family = 1;
    string download_location_url = 2;
 }
 
-enum Compliance {
-   FIPS_1402 = 0;
-}
+// Build maps 1:1 with Family
```

Comment:
Q: Do we really consider a build maps 1 by 1 to an OS family? I'm wondering the case that we change a build for each distribution version to handle diffs of C++ standard libraries a distribution has. @lizan can help this?

Reply:
I discussed this a little with @lizan the expectation is libc's map 1:1 with OS families. Will defer to his opinion on what this should look like though.

Reply:
No, it maps more to OS version, i.e. the "Ubuntu binary" should be able to run on RHEL 8 but not RHEL 7 / CentOS 7. btw I'm leaning toward build only one Linux binary that works for all our supported distros.

Reply:
Ok, what noun would you suggest for this mapping? I guess we don't have to expose any on this to a user via the CLI so it could just be glibc versions?